src/main/kotlin/no/nav/helsearbeidsgiver/pdl/PdlClient.kt

```diff
@@ -12,13 +12,13 @@ import io.ktor.http.contentType
  * Enkel GraphQL-klient for PDL som kan enten hente navn fra aktør eller fnr (ident)
  * eller hente mer fullstendig data om en person via fnr eller aktørid (ident)
  *
- * Authorisasjon gjøres via den gitte Token prodvideren, og servicebrukeren som er angitt i token provideren må være i
- * i AD-gruppen `0000-GA-TEMA_SYK` som dokumentert [her](https://pdldocs-navno.msappproxy.net/intern/index.html#_konsumentroller_basert_p%C3%A5_tema).
```

Comment:
Denne lenken fungerer ikke lenger og jeg finner ingenting tilsvarende i nåværende dokumentasjon. PDL-gjengen har sagt de vil gå bort ifra tilgang basert på tema, så dette virker ikke relevant lenger.

Reply:
Men vi kan fortsette å sende med tema selv om det ikke blir brukt til noe?

Reply:
Tema blir brukt i dag, men det skal fases ut til fordel for behandlingsnummer.

Fra [PDLs slack-kanal](https://nav-it.slack.com/archives/C020Q1K6EAY/p1685968467688529): 
> Vi ser at en del har tatt i bruk behandlingsnummer headeren og vi syns dette er veldig fint, det tar oss ett skritt nærmere å fase ut den tema baserte tilgangsstyringen.